VoxelSniperCore/src/main/java/com/github/kevindagame/voxelsniper/events/player/EventResult.java

```diff
@@ -0,0 +1,7 @@
+package com.github.kevindagame.voxelsniper.events.player;
+
+public enum EventResult {
+    ALLOW,
+    DEFAULT,
```

Comment:
When is this third state used? Or was it just added to future-proof?
What does Default mean?
Might be nice to add some documentation here.

Reply:
I have copied these states from the bukkit api.
Default just means that no listener has changed the state